### PR TITLE
docs: Update interacting with the Candid UI

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
**Overview**
Update the How to section because working with the Candid UI is different in 0.7.0.
Fix a missing link.

### CODE REVIEW

File: docs/modules/candid-guide/pages/candid-howto.adoc [hunk 1]
```diff
@@ -56,19 +55,50 @@ For more information about using `+dfx+` and the `+dfx canister call+` command,
...
+The command displays the canister identifier for the Candid UI with output similar to the following:
++
+....
+r7inp-6aaaa-aaaaa-aaabq-cai
+....
+. Start the {IC} locally by running the following command:
++
+[source,bash]
+----
+dfx start --background
```
Comment (Eve832) [inline docs/modules/candid-guide/pages/candid-howto.adoc:76]: --background is still buggy and almost always fails for me after 5-10 min.

File: docs/modules/candid-guide/pages/candid-howto.adoc [hunk 2]
```diff
@@ -56,19 +55,50 @@ For more information about using `+dfx+` and the `+dfx canister call+` command,
...
-. Add the `+candid+` endpoint to access the Candid web interface followed by the required `canisterId` parameter and canister identifier.
+. Add the required `canisterId` parameter and the Candid UI canister identifier returned by the `+dfx canister id+` command.
++
+For example, the full URL should look similar to the following but with the `+CANDID-UI_IDNETIFIER+` that was returned by the `+dfx canister id+` command:
++
+....
+http://127.0.0.1:8000/candid?canisterId=<CANDID-UI-IDENTIFIER>
+....
++
+The browser displays a form ior you to specify a canister identifier or choose a Candid description (`+.did+`) file.
```
Comment (Eve832) [inline docs/modules/candid-guide/pages/candid-howto.adoc:89]:
```suggestion
The browser displays a form for you to specify a canister identifier or choose a Candid description (`+.did+`) file.
```

File: docs/modules/candid-guide/pages/candid-howto.adoc [hunk 3]
```diff
@@ -56,19 +55,50 @@ For more information about using `+dfx+` and the `+dfx canister call+` command,
...
++
+
+image:candid-ui-select-id.png[]
++
+If you aren't sure of the canister identifier to use, you can run the `+dfx canister id+` command to look up the identifier for a specific canister name.
```
Comment (Eve832) [inline docs/modules/candid-guide/pages/candid-howto.adoc:94]:
```suggestion
If you aren't sure what canister identifier to use, you can run the `+dfx canister id+` command to look up the identifier for a specific canister name. 
```

**Review by Eve832 — CHANGES_REQUESTED** [review]

File: docs/modules/candid-guide/pages/candid-howto.adoc (hunk 1 above)
Comment (lsgunnlsgunn) [reply docs/modules/candid-guide/pages/candid-howto.adoc:76]: buggy, meaning it doesn't stay in the background?

Comment (Eve832) [reply docs/modules/candid-guide/pages/candid-howto.adoc:76]: correct.

**Review by Eve832 — APPROVED** [review]

File: docs/modules/candid-guide/pages/candid-howto.adoc (hunk 1 above)
Comment (chenyan-dfinity) [reply docs/modules/candid-guide/pages/candid-howto.adoc:76]: I don't think we need to mention `dfx start` here.  The whole tutorial assumes there is a replica running. Let me do a pass on this, and add instructions for accessing from the mainnet.

Comment (lsgunnlsgunn) [reply docs/modules/candid-guide/pages/candid-howto.adoc:76]: I would like to have instructions for using the Candid UI both locally (`dfx start` required) and running on `--network ic` with the new `_` URL

**Review by Eve832 — APPROVED** [review]

**Pull request merged**